client/src/components/LoginForm.jsx

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import { doLogin, changeLoginField } from '../actions/account';
+
```

Comment:
this is the next file. both are essentially the same I'm just unsure what they are actually doing.